client.go

```diff
@@ -693,7 +693,7 @@ type ReqSync struct {
 	Since           string
 	FilterID        string
 	FullState       bool
-	SetPresence     event.Presence
+	SetPresence     ReqPresence
```

Comment:
This change seems pointless

Reply:
oh yeah 🥴
I thought it was related to setpresence for some reason